src/ewmh_m2m/__main__.py

```diff
@@ -25,8 +25,10 @@ def move_to_screen(args):
     ))
     _logger.debug("Containing screen: %s", containing_screen)
 
-    window_state = window.maximized
+    window_maximized_state = window.maximized
+    window_fullscreen_state = window.fullscreen
     window.maximized = (False, False)
+    window.fullscreen = 0
```

Comment:
You forgot to rollback this part of your previous commit.
Fix this (using a bool) and it should be ready to merge.